cypress/integration/podcast.js

```diff
@@ -0,0 +1,125 @@
+const PODCAST_SLUG = '/podcasts/this-is-the-mongodb-podcast';
+const PROD_PODCAST_URL = `https://developer.mongodb.com${PODCAST_SLUG}`;
+
+const PODCAST_TITLE = 'This is the MongoDB Podcast';
+const PODCAST_DESCRIPTION =
```

Comment:
Was the formatting preventing us from testing the description on the page UI itself?

Reply:
It was. Have tried to format the text, but it's not help.

Reply:
It was, but have solution already.